docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
       - redis
     volumes:
       - ./app:/app
-    command: bash -c "/app/bin/wait-for-it.sh db:5432 -- celery -A experimenter beat -l debug"
+    command: bash -c "/app/bin/wait-for-it.sh db:5432 -- celery -A experimenter beat --pidfile ./celerybeat.pid -s ./celerybeat-schedule -l debug"
```

Comment:
I think that will put the files in the workdir, which is /app, which will then place them on the host filesystem, which is the opposite of what we want.  Try putting them right at the root path like `/celerybeat.pid` `/celerybeat-schedule`?

Reply:
my bads, did not completely understand... i think my new commit is what is originally asked?